Documentation/DebugJavascriptDevTools.md

```diff
@@ -0,0 +1,43 @@
+# JsRuntimeHost Script Debugging
+You can use Chromium DevTools to debug your JavaScript while running with the V8 JavaScript engine (Default Engine for Android).
+
+## Add the Remote Target URL
+Depending on which Chromium-based browser you're using (Edge or Chrome), navigate to `edge://inspect` or `chrome://inspect`.
```

Comment:
I know you copied this but a cross-browser way to do this is to use `about://inspect`.